docs/data/base/components/slider/UnstyledSliderIntroduction/css/index.tsx

```diff
@@ -124,13 +120,14 @@ function Styles() {
       box-shadow: 0 0 0 0.25rem ${alpha(isDarkMode ? cyan[300] : cyan[200], 0.5)};
     }
 
-    .CustomSlider-thumb:focus-visible {
+    .CustomSlider-thumb.${sliderClasses.focusVisible} {
```

Comment:
It looks to me that the Slider suffers from the same problem as the Switch - the `.Mui-focusVisible` class is not applied to the same element as `:focus-visible`. Since `.Mui-focusVisible` is just a workaround and is meant to be removed at some point, these two components will break.
While the fix in the docs is OK for now, we should open a new issue, similar to #34986 to fix this.

Reply:
I'll open an issue 🙌

Reply:
Already opened by Danilo - https://github.com/mui/material-ui/issues/40660